element.js

```diff
@@ -4,3 +4,6 @@ export const $todoList = document.getElementById("todo-list");
 export const $errorMessageElement = document.querySelector(".error-message");
 export const $searchInput = document.getElementById("search-input");
 export const $searchButton = document.getElementById("search-button");
+export const $allTodoButton = document.getElementById("all");
+export const $incompleteTodoButton = document.getElementById("incomplete");
+export const $completeTodoButton = document.getElementById("complete");
```

Comment:
`all`, `incomplete` and `complete` are very vague / general terms for id. ID should be specific and unique
```suggestion
export const $allTodoButton = document.getElementById("all-filter-button");
export const $incompleteTodoButton = document.getElementById("incomplete-filter-button");
export const $completeTodoButton = document.getElementById("complete-filter-button");
```